src/display/annotation_layer.js

```diff
@@ -990,6 +991,7 @@ class WidgetAnnotationElement extends AnnotationElement {
     if (this.data.alternativeText) {
       this.container.title = this.data.alternativeText;
     }
+    this.containter.removeAttribute("tabIndex");
```

Comment:
There's a pretty obvious typo here, that ought to throw; did you forget to test this patch locally :-P
Besides, assuming that the comment above is implemented this line should no longer be necessary.